README.md

```diff
@@ -420,6 +426,47 @@ Style: 0 / 0
 Total: 100 / 100
 ```
 
+##### Resubmitting a Student's Code by ID
+
+Proxy resubmit a student's code by their previous submission ID by using the `autograder.cli.courses.assignments.submissions.proxy.resubmit` command.
```

Comment:
Same as in the last section.
This is just a little abrupt.
You can soften it up how you choose.